pyintegration/deephaven2/_wrapper_abc.py

```diff
@@ -11,7 +11,16 @@
 
 
 class JObjectWrapper(ABC):
+    def __init_subclass__(cls, *args, **kwargs):
+        required_cls_attr = "j_object_type"
+        if not hasattr(cls, required_cls_attr):
+            raise NotImplementedError(
+                f"Class {cls} lacks required `{required_cls_attr}` class attribute"
+            )
+        if not isinstance(getattr(cls, required_cls_attr), type):
```

Comment:
Where is `type` coming from?

Reply:
the type class of Python, the root meta class.